README-prerequisite.md

```diff
@@ -0,0 +1,39 @@
+# Alfresco Digital Business Platform Prerequisites
+
+The Alfresco Digital Business Platform Deployment requires:
+
+| Component   | Recommended version | Getting Started Guide |
+| ------------|:-----------: | ---------------------- |
+| Docker      | 18.09.0      | https://docs.docker.com/ |
+| Kubernetes  | 1.10.3       | https://kubernetes.io/docs/tutorials/kubernetes-basics/ |
+| Kubectl     | 1.10.3       | https://kubernetes.io/docs/tasks/tools/install-kubectl/ |
+| Helm        | 2.9.1        | https://docs.helm.sh/using_helm/#quickstart-guide |
+| Homebrew    | 1.7.7        | https://brew.sh/            |
+
+In addition, since it can be deployed to different environments, check the requirements for:
+- [KOPS deployment](#kops)
+- [EKS deployment](#eks)
+- [Docker for Desktop deployment](##docker-for-desktop---mac)
+
+# KOPS
+For KOPS deployment you will also need:
+
+| Component   | Recommended version | Getting Started Guide |
+| ------------|:-----------: | ---------------------- |
+| Kops        | 1.8.1        | https://github.com/kubernetes/kops/blob/master/docs/aws.md |
+
+# EKS
+For EKS deployment you will also need:
+
+| Component   | Recommended version | Getting Started Guide |
+| ------------|:-----------: | ---------------------- |
+| AWS Cli     | 1.16.33      | https://github.com/aws/aws-cli#installation |
+
+# Docker for desktop - mac
+For Docker for desktop on mac deployment you will also need:
+
+| Component   | Recommended version | Getting Started Guide |
+| ------------| :-----------: | ----------------------      |
+| Homebrew    | 1.7.7         | https://brew.sh/            |
```

Comment:
Do we need this here and in the DBP prerequisites above?

Reply:
Nope, I moved it up and forget to remove it from here.